docs/en/01_Managing_your_website/99_How_to_find_CMS_version.md

```diff
@@ -3,12 +3,8 @@ summary: How to find out the version of SilverStripe CMS you are using.
 
 # Finding out your version of SilverStripe CMS
 
-It's good to know the SilverStripe CMS version you are using to manage your content to ensure you are using the correct User help guides.
+It's good to know the SilverStripe CMS version you are using to manage your content to ensure you are using the correct User help guides. The CMS help menu can be found in the Action toolbar of the CMS menu and provides visibility of the CMS version number. It is directly visible in the menu and shows the full version details on hover.
```

Comment:
Do you think it might be worth adding e.g. "since 4.2" or some other way of making explicit the differences between versions (in that the user is trying to find out which one in the first place) - if e.g. 4.0 is installed then the mouse hover is still relevant - I'm not 100% sure that the docs are that granule in their splitting of versions.

This is to say that I think this update will be the only version of the docs, I think we should consider people lagging behind on old versions too though.

Reply:
Suggestion text @sachajudd 

The CMS version can be found in the help menu (SilverStripe logo) at the bottom of the CMS menu. You can view the CMS version number, with full version details showing on hover. This functionality is new as of SilverStripe 4, in earlier versions hover over the SilverStripe logo at the top of the CMS menu to access CMS version details. 

Knowing the SilverStripe CMS version is helpful to ensure you use the correct User help guides, to ensure compatibility of modules, and see what features are apart of certain release.